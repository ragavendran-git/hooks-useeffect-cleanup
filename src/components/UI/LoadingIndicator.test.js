import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from "enzyme-to-json";
import React from 'react';

import LoadingIndicator from './LoadingIndicator';

configure({ adapter: new Adapter() });

it("Should render correctly", () => {
    const wrapper = shallow(<LoadingIndicator />);
    expect(toJson(wrapper)).toMatchSnapshot();
});