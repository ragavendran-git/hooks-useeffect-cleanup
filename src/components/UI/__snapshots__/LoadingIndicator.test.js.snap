// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<div
  className="lds-ring"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
